lib/nostrum/api.ex

```diff
@@ -310,10 +320,7 @@ defmodule Nostrum.Api do
 
   ## Examples
 
-      Nostrum.Api.delete_own_reaction(123123123123, 321321321321, "\xF0\x9F\x98\x81")
-      
-      Nostrum.Api.delete_own_reaction(123123123123, 321321321321, URI.encode("\u2b50"))
-      
+  See `create_reaction/3` for similar examples.
```

Comment:
I don't think we need the `Examples` sub-header if we're going to point to another function like this.

Reply:
Removed in latest commit.